lib/response.js

```diff
@@ -155,16 +155,26 @@ function getNullSocket () {
 }
 
 function copyHeaders (response) {
-  response._lightMyRequest.headers = Object.assign({}, response.getHeaders())
+  const headers = Object.assign({}, response.getHeaders())
+
+  for (const [name, value] of Object.entries(headers)) {
+    if (Array.isArray(value)) {
+      headers[name] = value.map(value => '' + value)
+    } else {
+      headers[name] = '' + value
+    }
+  }
```

Comment:
I don't think this should be done inside `copyHeaders`, but rather right before generating the payload: https://github.com/fastify/light-my-request/pull/248/files#diff-c03f99ad1a5cf68a9b32468371973a5898c4c9c9d5f097e124f1d2b7b27db11dR118.

The reason is that if multiple `res.setHeader()` are issued, they will not be picked up.